### PR TITLE
feat(signup): add install metamask guide

### PR DESCRIPTION
**Pull request by Jasmine-liang**

### CODE REVIEW

File: src/components/Forms/WalletAuthForm/Select.tsx [hunk 1]
```diff
@@ -211,6 +211,26 @@ const Select: React.FC<FormProps> = ({
...
+        ) : (
+          <Form.List.Item
+            title={
+              <TextIcon
+                color="black"
+                icon={<IconMetaMask24 size="md" />}
+                size="md"
+                spacing="xtight"
+              >
+                Install MetaMask
```
Comment (robertu7) [inline src/components/Forms/WalletAuthForm/Select.tsx:223]: translation?

File: src/common/enums/externalLinks.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export const EXTERNAL_LINKS = {
     'https://docs.like.co/v/zh/user-guide/likecoin-button/superlike?utm_source=Matters&utm_medium=website&utm_campaign=superlike_funnel',
   PLANET: 'https://www.planetable.xyz/',
   ENS_DOCS: 'https://docs.ens.domains/',
+  METAMASK: 'https://metamask.io/',
```
Comment (robertu7) [inline src/common/enums/externalLinks.ts:20]: direct to https://metamask.io/download/?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/common/enums/externalLinks.ts (hunk 2 above)
Comment (devformatters) [reply src/common/enums/externalLinks.ts:20]: Yes, go to the `download` page

File: src/components/Forms/WalletAuthForm/Select.tsx (hunk 1 above)
Comment (devformatters) [reply src/components/Forms/WalletAuthForm/Select.tsx:223]: - 安裝 MetaMask
- 安装 MetaMask
- Install MetaMask

**Review by robertu7 — APPROVED** [review]

**Pull request merged**